src/ui/layout/BrowseBar.vue

```diff
@@ -179,7 +179,8 @@ export default {
       });
     },
     hasParent() {
-      return this.domainObject !== PLACEHOLDER_OBJECT && this.parentUrl !== '/browse';
+      return false;
+      // return this.domainObject !== PLACEHOLDER_OBJECT && this.parentUrl !== '/browse';
```

Comment:
What's the thinking behind this change?

Reply:
@scottbell to look at this

Reply:
It's because in `mct-tree.vue`, we call [calculateHeights](https://github.com/nasa/openmct/blob/vue3-compat-migration/src/ui/layout/mct-tree.vue#L304) which [loops forever](https://github.com/nasa/openmct/blob/vue3-compat-migration/src/ui/layout/mct-tree.vue#L942C14-L942C14) looking for a tree height. Changing `calculateHeights` to fire post the first Vue tick resolves this issue (and reveals more problems 😀)

Reply:
Fixed (and pushed) after also [toRaw'ing the proxied domainObject](https://github.com/nasa/openmct/blob/vue3-compat-migration/src/ui/layout/BrowseBar.vue#L183) in BrowseBar so the comparison would work.